### PR TITLE
Added docs and repo config for the aztools modules

### PR DESCRIPTION
**Pull request by mikefrobbins**
# PR Summary

Added docs and repo config for the aztools modules

## PR Checklist

- [x] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [x] **Summary:** This PR's summary describes the scope and intent of the change.
- [x] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [x] **Style:** This PR adheres to the [style guide][style].

[contrib]: https://learn.microsoft.com/powershell/scripting/community/contributing/overview
[style]: https://learn.microsoft.com/powershell/azure/azps-docs-style-guide

### CODE REVIEW

File: docs-conceptual/aztools/overview.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ms.topic: overview
+title: Az Tools PowerShell modules overview
+---
+
+# Az Tools PowerShell modules
+
+The Azure PowerShell team has created a set of tool modules that extend the capabilities of Azure
+PowerShell. These modules can be installed from the PowerShell Gallery and loaded on demand. The
+modules aren't officially part of Azure PowerShell, but the Azure PowerShell product team maintains
+them. The level of support for each module varies. For more details, see the landing page for each
```
Comment (sdwheeler) [inline docs-conceptual/aztools/overview.md:15]:
```suggestion
them. The level of support for each module varies. For more details, see the overview page for each
```

File: docs-conceptual/aztools/predictor-overview.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+Az.Tools.Predictor is tailored to the user's current context, making its recommendations highly
+relevant and time-saving.
+
+### Getting started with Az.Tools.Predictor
+
+To begin using Az.Tools.Predictor, users must ensure they have installed the correct PowerShell and
+PSReadLine version. Following this, the Az.Tools.Predictor module can be installed and enabled for
+all sessions with simple commands. The module supports two view modes for suggestions: an 'inline
+view', which is the default showing one suggestion at a time, and a 'list view', which can be
+toggled using the F2 key.
```
Comment (sdwheeler) [inline docs-conceptual/aztools/predictor-overview.md:34]: How do you install it?

**Review by sdwheeler — COMMENTED** [review]
The cmdlet reference is rough and needs formatting, but it will work.

See the two comments I made.

File: docs-conceptual/aztools/predictor-overview.md (hunk 2 above)
Comment (mikefrobbins) [reply docs-conceptual/aztools/predictor-overview.md]: Added installation info

**Pull request merged**